DotaPlayerData.API/OpenDotaApiClient.cs

```diff
@@ -58,9 +58,24 @@
~        
~            return await response.GetStringAsync();
~        }
~        catch (FlurlHttpException e)
~        {
             throw;
         }
     }
+
+    public async Task<string> SearchPlayer(string name)
+    {
+        try
+        {
+            string searchEndpoint = _baseUri.AppendPathSegment("search").SetQueryParam("q", name);
+            var response = await searchEndpoint.GetAsync();
+
+            return await response.GetStringAsync();
+        }
+        catch (FlurlHttpException e)
+        {
+            throw;
+        }
```

Comment:
**Suggestion:** Consider using `ConfigureAwait(false)` in your async calls to avoid potential deadlocks in the UI thread.
```suggestion
    public async Task<string> SearchPlayer(string name)
    {
        try
        {
            string searchEndpoint = _baseUri.AppendPathSegment("search").SetQueryParam("q", name);
            var response = await searchEndpoint.GetAsync().ConfigureAwait(false);

            return await response.GetStringAsync().ConfigureAwait(false);
        }
        catch (FlurlHttpException e)
        {
            throw;
        }
    }
```